language/en/acp/board.php

```diff
@@ -570,6 +570,7 @@
 	'SMTP_PORT'						=> 'SMTP server port',
 	'SMTP_PORT_EXPLAIN'				=> 'Only change this if you know your SMTP server is on a different port.',
 	'SMTP_SERVER'					=> 'SMTP server address',
+	'SMTP_SERVER_EXPLAIN'			=> 'Note that you have to provide the protocol that your server uses. If you are using SSL, this has to be "ssl://your.mailserver.com"',
```

Comment:
Im ersten Satz nutzt der Server ein Protokoll, im zweiten jedoch der Benutzer. "If you are using SSL" bezieht sich daher eigentlich genaugenommen auf "your server" - nur ist SSL kein Server. Es müsste wohl "If your server is using..." oder "If you are using a server using..." heißen.